app/models/user.rb

```diff
@@ -51,6 +52,10 @@ class User < ApplicationRecord
     self.name = self.handle if self.name.blank?
   end
 
+  after_create_commit do
+    create_communication_preferences
```

Comment:
This ensures that after the record is created, a communication preferences record is also created?

Reply:
Correct, yes.